airflow/scripts/git_scripts/aggregate_repo_activity.py

```diff
@@ -0,0 +1,85 @@
+
+import requests
+from collections import defaultdict
+from datetime import datetime, timedelta
+
+token = 'put_token_here'   # add your token
```

Comment:
Change the token to be accepted from a connection in Airflow instead of using a local token (for security reasons).

Reply:
Is this how to manage the token to connect with Airflow 

token = os.getenv('AIRFLOW_GITHUB_TOKEN')
if token is None:
    raise ValueError("AIRFLOW_GitHUB_TOKEN environment variable must be set.")

Reply:
You have two ways to handle passwords and tokens:

1. If you are running the airflow docker :

- connect the UI using localhost:8080
- Go to connections : 
![image](https://github.com/eyalFischel/maakaf-stats/assets/39159613/95a79c8c-5bc7-443c-ac4d-85840506b8c5)
- Add the connection : 
![image](https://github.com/eyalFischel/maakaf-stats/assets/39159613/e59a94b6-9a6c-4ee1-aa75-dcd95c87f875)
- Call it from your script using : 
![image](https://github.com/eyalFischel/maakaf-stats/assets/39159613/690e386c-ad95-43bd-bf39-ee6d3c83e2e0)

2. Use .env file (if you are running it manually or with corn job):
- `pip install python-dotenv`

- `from dotenv import load_dotenv
import os

# Load the .env file
load_dotenv()

# Access the environment variables
git= os.getenv('git_token')